.github/workflows/try.yml

```diff
@@ -2,10 +2,18 @@ name: CI
 
```

Comment:
This is not CI, but a workflow to deploy docs (in case it would be CD, but borderline).

Call it properly, something like `Deploy Sphinx documentation`.

Reply:
Change also file name, to match the descriptive workflow name.

Thus, following the example, you could call it `deploy-sphinx-docs-to-ghpages`.